sirepo/pkcli/admin.py

```diff
@@ -24,6 +24,27 @@
 import shutil
 
 
+def add_jupyterhub_role():
```

Comment:
This is not something we will execute manually so I don't think it should be here. It needs to be a db upgrade so that it gets run on all implementations that implement jupyterhublogin similar to _migrate_db_file. I think we would check to see if juptyerhublogin is enabled and if there are any users with that role. If the answer is none, then we need to run the ugprade, atomically at startup.

Reply:
252b9c1d